### PR TITLE
add: `unicons-git`

### PR DESCRIPTION
**Pull request by ajstrongdev**
Create the unicons pacscript.

### CODE REVIEW

File: packages/unicons-git/unicons.pacscript [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+source=("https://github.com/rhino-linux/unicons.git")
+arch=("all")
+pkgver="0.0.1"
+gives="unicons"
+pkgdesc="Unicorn Desktop Icons. By the Rhino Linux Team"
+maintainer=("AJStrong <ajstrongdev@proton.me>")
+depends=("gtk-update-icon-cache")
+
+build() {
+  cd "${gives}"
```
Comment (Elsie19) [inline packages/unicons-git/unicons.pacscript:11]:
```suggestion
```

File: packages/unicons-git/unicons.pacscript [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  mkdir -p "${pkdir}/usr/share/icons"
+}
+
+package() {
+  cd "${gives}"
+  install -dm755 "${pkgdir}/usr/share/icons/unicons"
```
Comment (Elsie19) [inline packages/unicons-git/unicons.pacscript:17]: You aren't installing anything, you're just making a directory.

**Review by Elsie19 — CHANGES_REQUESTED** [review]

File: packages/unicons-git/unicons.pacscript
Comment (Elsie19) [inline packages/unicons-git/unicons.pacscript:1]: Wrong pacscript name.

File: packages/unicons-git/unicons-git.pacscript [hunk 3]
```diff
@@ -7,12 +7,11 @@ pkgdesc="Unicorn Desktop Icons. By the Rhino Linux Team"
...
+prepare() {
   mkdir -p "${pkgdir}/usr/share/icons/Unicons/"
-  cp -r unicons/* /usr/share/icons/Unicons
 }
 
-post_install() {
-  gtk-update-icon-cache "/usr/share/icons/Unicons"
+package() {
+  cp -r "${gives}"/* "${pkgdir}/usr/share/icons/Unicons"
+  gtk-update-icon-cache "${pkgdir}/usr/share/icons/Unicons"
```
Comment (ajstrongdev) [inline packages/unicons-git/unicons-git.pacscript:16]: have been told by elsie to make this `post_install()` why change?

Comment (oklopfer) [reply packages/unicons-git/unicons-git.pacscript:16]: because elsie didnt understand why it was failing.

Comment (ajstrongdev) [reply packages/unicons-git/unicons-git.pacscript:16]: understandable, have a nice day

**Review by Elsie19 — APPROVED** [review]

**Pull request merged**